app/src/main/kotlin/com/wire/android/ui/theme/Typography.kt

```diff
@@ -0,0 +1,81 @@
+package com.wire.android.ui.theme
+
+import androidx.compose.material.Typography
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.sp
+
+
+val Typography.subLine1: TextStyle
+    get() = TextStyle(
+        color = WireColor.Dark70Gray,
```

Comment:
Typography shouldn't contain colors - `typography`, `colors` and `shapes` are three separate elements of `MaterialTheme` and I think they shouldn't be mixed

Typography in my opinion should only specify the parameters of the font (weight, size,  letter spacing) and then colors should be set separately, depending on the style of the app, thanks to that approach, we can define a typography for a specific text and let our Theme decide if it should have light or dark coloured text

Reply:
will leave it as it is for now, as we are going to refactor it soon